vector/src/main/java/im/vector/app/core/platform/VectorBaseActivity.kt

```diff
@@ -105,7 +105,7 @@ abstract class VectorBaseActivity<VB : ViewBinding> : AppCompatActivity(), Maver
     protected val viewModelProvider
         get() = ViewModelProvider(this, viewModelFactory)
 
-    protected fun <T : VectorViewEvents> VectorViewModel<*, *, T>.observeViewEvents(observer: (T) -> Unit) {
+    fun <T : VectorViewEvents> VectorViewModel<*, *, T>.observeViewEvents(observer: (T) -> Unit) {
```

Comment:
Why this change?

Reply:
unfortunately the current visibility means we can only observe `ViewModel` `ViewEvents` from within an activity which is a bit too limiting, it's needed by https://github.com/vector-im/element-android/pull/4656/files#diff-98284f9358684c3751933502d694e1babd0a0b7038302282cd0f2037adb25970R109

I could copy the logic to avoid the visibility change but there's already some other public functions in the base activity so I thought this change wouldn't be _toooo_ out of place

Reply:
OK